### DISCUSSION TITLE
Issue i cant add my school email.

### DISCUSSION THREAD

**Discussion by LUPLUV** [community/community#39345] [GitHub Education]
Hey there,
i just tried to get the Students Developer Pack, but its not working because:
1. I cant add my school email to my account because our school email server is blocking mails from different email servers.
2. If i search for my school the mail name @hlg-fuert.de comes up, but thats only for the teachers, our student mails end with @hlg-netz.de
I hope you can help me.

EDIT:
I tried it with my report card, but thats not working too

**Comment by kodepirate** [comment]
Hey @LUPLUV  I would recommend you to try it with your fee structure/bona fide  or the money receipt of your admission in your college.
It will work

**Reply by LUPLUV** [reply]
Im going to School in germany, i dont have to pay for school

**Comment by kodepirate** [comment]
Then show them something authorised by your school
Something that proves that you read in that school @LUPLUV

**Comment by samus-aran** [comment]
Hi @LUPLUV 

That's unfortunate you aren't able to use your school email. Are you able to use any other email?

Let us know how you get on.
